docs/content/recipes/interoperability/identifier-mapping.md

```diff
@@ -0,0 +1,384 @@
+
+The following is an incomplete list of identifier mapping services.
+
+* [bridgedb.org](https://bridgedb.org/)
+
+    > [BridgeDb](https://bridgedb.org/) is a framework for identifier mapping within the life sciences which covers genes, proteins, genetic variants, metabolites, and metabolic reactions. It is provided as a web service, a standalone application that can be installed locally, a Java library or an R Package.
+    > 
+    > It permits users to lookup equivalent database identifiers for a given database identifier within a specified organism. The following `curl` command to the REST API retrieves the equivalent identifiers for the EntrezGene (now known as NCBI Gene) `L` identifier `1234` for the `Human` gene [CCR5](https://www.ncbi.nlm.nih.gov/gene/1234) as a TSV file.
+    > ```
+    > curl -X GET "https://webservice.bridgedb.org/Human/xrefs/L/1234" -H "accept: */*"
+    > ```
+    > For more details about using BridgeDb as an identifier mapping service, please see [this recipe]XXX.
+
+* [UniChem](https://www.ebi.ac.uk/unichem/)
+    > [UniChem](https://www.ebi.ac.uk/unichem/) is a specialised identifier mapping service for chemical structures. For a chemical structure -- specified as an identifier, InChI, or InChI Key -- it will equivalent structures found in the [EMBL-EBI](https://www.ebi.ac.uk/) chemistry resources.
+    > 
+    > The following `curl` command retrieves the equivalent database identifiers for the ChEMBL identifier `CHEMBL12` [DIAZEPAM](https://www.ebi.ac.uk/chembl/compound_report_card/CHEMBL12/) and returns the result as a JSON object.
+    > ```
+    > curl -X GET "https://www.ebi.ac.uk/unichem/rest/src_compound_id/CHEMBL12/1" -H "accept: */*"
+    > ```
+    > For more information including the available methods, see the [UniChem REST documentation](https://www.ebi.ac.uk/unichem/info/webservices).
+* [sameas.org](https://sameas.org/)
+  
+    > [sameas.org](https://sameas.org/) is a general purpose service that will return a set of equivalent URLs for a given URL. The equivalences are based on an incomplete set of `owl:sameAs` statements contained in data available on the web.
+    > 
+    > The following `curl` command retrieves the equivalent URLs for EBI RDF Platform representation of ChEMBL [DIAZEPAM](https://rdf.ebi.ac.uk/resource/chembl/molecule/CHEMBL12) as a JSON object.
+    > ```
+    > curl -iLH "Accept: application/json" "http://sameas.org/?uri=http://rdf.ebi.ac.uk/resource/chembl/molecule/CHEMBL12"
+    > ```
+    > Note that the coverage of sameas.org within the life sciences is very small.
+
+
+* [Identifiers.org](http://identifiers.org/)
```

Comment:
httpS://identifiers.org/ seems to be the preferred link now.

Reply:
Done